src/settings/mod.rs

```diff
@@ -130,11 +130,18 @@ impl Settings {
         }
 
         config.merge(File::with_name("config/local").required(false))?;
-
         config.merge(Environment::with_prefix("fxa_email"))?;
 
         match config.try_into::<Settings>() {
             Ok(settings) => {
+                if let Ok(rocket_env) = env::var("ROCKET_ENV") {
+                    if rocket_env == "production"
+                        && &settings.message_id_hmac_key == "YOU MUST CHANGE ME"
```

Comment:
I'm thinking here it would be best if we actually checked that the HMAC was valid, instead of just checking for `YOU MUST CHANGE ME`. But I remember us talking in a meeting about doing it this way, so let me know what you guys think :)

Reply:
> 'm thinking here it would be best if we actually checked that the HMAC was valid

Sure let's do that if possible, and we also need a sample one for `dev.json` right?

Reply:
When something is not set in `dev.json` it defaults to `default.json` so in there we would just get `YOU MUST CHANGE ME`. Do you think we should have something else for `dev.json`?

Reply:
from mtg: So these can be any string so let's keep it this way